roles/skydive/tasks/provision.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Login As Super User
+  shell: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+    and admin_user is defined
+    and admin_password is defined
+
+- name: Check if namespace "{{ skydive_namespace }}" exists
+  shell: "kubectl get ns | grep -w {{ skydive_namespace }} | awk \'{ print $1 }\'"
+  register: ns
+
+- name: Create {{ skydive_namespace }} namespace on the kubernetes cluster
+  shell: kubectl create namespace {{ skydive_namespace }}
+  when: ns.stdout != skydive_namespace
+    and cluster=="kubernetes"
+
+- name: Create {{ skydive_namespace }} project on the openshift cluster
```

Comment:
this task is redundant.  `kubectl create namespace` will accomplish the same thing

Reply:
I think It's not the same because here the project is related to the user that makes the login into the cluster.

Reply:
Projects are namespaces++.  They have some resource constraint capabilities and a few other things.  If you explicitly are using something that a project offers out of the box, then it's not redundant.  Most of the time, namespaces will cover what you need, so I wanted to make sure.